addons/token-exchange/manifests/spoke_deployment.yaml

```diff
@@ -31,6 +31,7 @@ spec:
           - "--hub-kubeconfig=/var/run/hub/kubeconfig"
           - "--cluster-name={{ .ClusterName }}"
           - "--namespace={{ .AddonInstallNamespace }}"
+          - "--handlers={{ .Handlers }}"
```

Comment:
This doesn't feel clean. We need to find alternatives.

Reply:
Updated to have mode instead of handlers